### PR TITLE
Prepull bazel 4.2.1 in bootstrap image

### PR DESCRIPTION
**Pull request by fgimenez**
/cc @dhiller @rmohr

Signed-off-by: Federico Gimenez <federico.gimenez@gmail.com>

### CODE REVIEW

File: images/bootstrap/Dockerfile [hunk 1]
```diff
@@ -99,8 +99,9 @@ RUN  curl -Lo ./bazelisk https://github.com/bazelbuild/bazelisk/releases/downloa
 # Cache the most commonly used bazel versions inside the image
 # and remove resulting cache directories to save a few hundred MB
 RUN USE_BAZEL_VERSION=3.4.1 bazel version && \
-  USE_BAZEL_VERSION=4.0.0 bazel version && \
-  rm -rf /root/.cache/bazel
+    USE_BAZEL_VERSION=4.0.0 bazel version && \
```
Comment (dhiller) [inline images/bootstrap/Dockerfile:102]: Should we maybe also prepull 4.1.0 which is used in project-infra IIRC?

**Comment by dhiller** [comment]
Great, thanks for this!

File: images/bootstrap/Dockerfile (hunk 1 above)
Comment (fgimenez) [reply images/bootstrap/Dockerfile]: Good catch, in order to keep the image size low I've changed the PR to only prepull the versions used in kubevirt and project-infra, PTAL

**Review by dhiller — APPROVED** [review]
/approve

Thanks!

**Pull request merged**